models/running_input.go

```diff
@@ -10,6 +10,7 @@ import (
 var (
 	GlobalMetricsGathered = selfstat.Register("agent", "metrics_gathered", map[string]string{})
 	GlobalGatherErrors    = selfstat.Register("agent", "gather_errors", map[string]string{})
+	GlobalTimeoutWarnings = selfstat.Register("agent", "gather_timeouts", map[string]string{})
```

Comment:
Sorry for being pedantic but can you please also fix the variable names!?

Reply:
I've redone the variable name and the associated functions.